staging/src/k8s.io/legacy-cloud-providers/gce/gce_disks.go

```diff
@@ -515,7 +515,7 @@ func (g *Cloud) GetLabelsForVolume(ctx context.Context, pv *v1.PersistentVolume)
 
 	// If the zone is already labeled, honor the hint
 	name := pv.Spec.GCEPersistentDisk.PDName
-	zone := pv.Labels[v1.LabelFailureDomainBetaZone]
+	zone := pv.Labels[v1.LabelTopologyZone]
```

Comment:
This is used by the admission controller? I think the pv object could be user-created, so we may need to still handle beta labels.